en/includes/guides/branding/customize-email-templates.md

```diff
@@ -8,10 +8,38 @@ Once you publish your [branding preferences]({{base_path}}/guides/branding/confi
 
 {{email_template_note}}
 
-The branding variables that affect the email templates are as follows:
+{% if product_name == "Asgardeo" %}
 
 !!! note "Email branding for B2B applications"
-    If you have [organizations]({{base_path}}/guides/organization-management/manage-organizations/) configured, note that the email branding you configure for your organization (root) also applies to your organizations.
+
+    If you have [organizations]({{base_path}}/guides/organization-management/manage-organizations/) configured,
```

Comment:
```suggestion
    If you have set up [organizations]({{base_path}}/guides/organization-management/manage-organizations/), these organizations automatically inherit the branding of the root organization.
```

Reply:
Fixed with https://github.com/wso2/docs-is/pull/5038/commits/edb4c27ab72cf145858824262f20bf40956ac591